NDTensors/test/dense.jl

```diff
@@ -107,13 +107,13 @@ using Test
   end
 
   @testset "Random constructor" begin
-    T = randomTensor(2, 2)
+    T = randomDenseTensor(2, 2)
```

Comment:
So are we getting rid of `randomTensor`?

Reply:
I would still keep it, it seems reasonable for `randomTensor` to default to `Dense` storage.

Reply:
I have added back randomTensor and have it use `default_storagetype`